code/API_definitions/device-status.yaml

```diff
@@ -8,7 +8,7 @@ info:
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
-  version: 0.5.0-wip
+  version: 0.6.0-wip
```

Comment:
Under description , we can mention "connectivity" too like "This API allows to query device roaming status, **connectivity** and subscribe to event related to device roaming status."